core/src/main/kotlin/riven/core/models/workflow/environment/WorkflowNodeContext.kt

```
@@ -0,0 +1,4 @@
+package riven.core.models.workflow.environment
+
+class WorkflowNodeContext {
+}
```

detekt (1.23.8)

[warning] The class or object WorkflowNodeContext is empty.

(detekt.empty-blocks.EmptyClassBlock)